Iced/Intel/DecoderInternal/OpCodeHandlers64.cs

```diff
@@ -2012,33 +2012,33 @@ sealed class OpCodeHandler_RegIb3 : OpCodeHandler {
 		}
 
 		static readonly (Code code, Register register)[] noRexPrefix = new(Code code, Register register)[8] {
-			(Code.Mov_AL_imm8, Register.AL),
```

Comment:
The Code value is, after the changes, always identical here and can be removed (same thing with info16/32/64 tables below)